docs/csharp/programming-guide/types/boxing-and-unboxing.md

```diff
@@ -25,7 +25,7 @@ The object `o` can then be unboxed and assigned to integer variable `i`:
 
 The following examples illustrate how boxing is used in C#.
 
-[!code-csharp[csProgGuideTypes#47](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsProgGuideTypes/CS/Class1.cs#47)]
+[!code-csharp[csProgGuideTypes#16](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsProgGuideTypes/CS/Class1.cs#16)]
```

Comment:
This is not the correct snippet. It's already included later in the article. The correct snippet is 47, but the problem is that the file contains two separate snippets with the same name/number. The correct fix is to make the numbering different, and potentially review references to both snippets to make sure they're correct.

@gewarren I'm expecting open publishing build to fail for duplicate snippet names. It might help if you could submit an internal feature request. If it's unlikely to get implemented, I'll see if I could add that to the GitHub Action when I get back to it.

Reply:
For reference, this is when the issue happened: https://github.com/dotnet/docs/pull/22185

Reply:
Good idea @Youssef1313 - I've opened an internal customer suggestion.